### ISSUE TITLE
new issue

### ISSUE THREAD

**Issue by wesdse001** [wesdse001/test#1]

**Comment by wesdse001** [comment]
@wesdse001 thank you for the reply. You are absolutely right.

**Comment by wesdse001** [comment]
@jing-c thank you for the reply. You are absolutely right.

**Comment by wesdse001** [comment]
🙏 @jing-c thank you for your reply. And you are absolutely right. Just by looking at the repo statistics alone, I saw that around 90 - 95% of the contributions are made by Google employees (my estimations are probably off haha). Therefore, financial support is most likely not an issue for those developers.

However, the **donate** button could be a way for other parties to show appreciation & support. For example, one of the many companies that use `tensorflow`, or even a YouTuber that teaches a course on this technology. Those parties could want to support `tensorflow`'s development in ways other than just engineering hours.

But, I can see where you are coming from. Because `tensorflow` is backed by Google, external financial support is not required. And perhaps Boost Lab is only tailored towards projects that does not have the backing of a tech giant.

Anyways, we appreciate your feedback. BTW, we tried out `tensorflow.js` and we love what Google is doing with the technology 👍.
